### DISCUSSION TITLE
Paste content is glitching.

### DISCUSSION THREAD

**Discussion by VincentLuders** [EpicenterHQ/epicenter#192] [Ideas]
Awesome app, I was trying to build it myself, but no need for it anymore. 

The one biggest thing why it's currently disrupting my workflow is that if you take longer audios and it types these text in the manner that it does it right now it's starting to become unreliable: It puts the letters occasionally in a wrong sequence. 

Instead, what is way faster and also 100% reliable is if you would merely control the paste function as soon as it lands into the clipboard.

Is that a good idea?

**Comment by braden-w** [comment]
Hey @VincentLuders , thanks for reaching out! Sorry for the delay, I didn't check the discussions for a while and was foucsed on issues.

This should be addressed in the latest release! See #509, which was merged last month.
